src/components/Progress/story.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import Progress from '.';
+import Box from '../Box';
+import { IconButton } from '../..';
+import { ChevronLeftIcon, ChevronRightIcon } from '../../assets';
+export const Demo = () => {
```

Comment:
nit: this could use an extra newline 
```suggestion

export const Demo = () => {
```